dev-aws/kafka-shared-msk/data-infra/data-infra.tf

```diff
@@ -10,7 +10,7 @@ resource "kafka_topic" "events" {
     "local.retention.ms" = "86400000"
     # allow max 1 MB for a message
     "max.message.bytes" = "1048576"
-    "compression.type"  = "zstd"
+    "compression.type"  = "gzip"
```

Comment:
how does changing the compression help debugging? Are things already `gzip` compressed so this does nothing?

Reply:
Yes, the topic we are reading from to mirror into msk is gzip - so I am trying to rule out compression for the errors I am currently seeing in mm2